src/python/gudhi/representations/vector_methods.py

```diff
@@ -574,3 +575,145 @@ def __call__(self, diag):
             numpy array with shape (**threshold**): output complex vector of coefficients.
         """
         return self.fit_transform([diag])[0,:]
+
+def _lapl_contrast(measure, centers, inertias, eps=1e-8):
```

Comment:
`eps` is homogeneous to a length here (I assume it is meant to avoid division by 0). If I am working with distances between atoms and write them in light years, `1e-8` is going to be huge. Is there a way to make it scale-invariant (use eps times the numerator, or compute a minimum after the quotient), or should we document that data needs to be normalized? (or maybe ATOL already normalizes data and I haven't seen it yet)

Reply:
I removed `eps`...